src/app/core/services/electron.service.ts

```diff
@@ -15,52 +13,22 @@ declare global {
   providedIn: 'root',
 })
 export class ElectronService {
-  readonly INSPECT_MENU_ITEM: MenuItem;
-  rightClickPosition = null;
-  rightClickMenu: Menu;
-
   ipcRenderer: typeof ipcRenderer;
-  remote: typeof remote;
-  clipboard: typeof clipboard;
-  fs: typeof fs;
 
   constructor() {
     if (this.isElectron()) {
       this.ipcRenderer = window.require('electron').ipcRenderer;
-      this.remote = window.require('electron').remote;
-      this.clipboard = window.require('electron').clipboard;
-      this.fs = window.require('fs');
-
-      this.INSPECT_MENU_ITEM = new this.remote.MenuItem({
```

Comment:
All usages of `remote` have been removed from `ElectronService`.
The functionality here (involving the "Inspect Element" feature, ability to open external links, ability to clear cookies) has been re-implemented in `main.ts` and `electron-utils/menu-bar.ts`